src/main/java/mate/academy/springboot/datajpa/model/Product.java

```diff
@@ -0,0 +1,30 @@
+package mate.academy.springboot.datajpa.model;
+
+import jakarta.persistence.CascadeType;
+import jakarta.persistence.Entity;
+import jakarta.persistence.FetchType;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.OneToOne;
+import jakarta.persistence.Table;
+import java.math.BigDecimal;
+import lombok.Getter;
+import lombok.Setter;
+import lombok.ToString;
+
+@Entity
+@Table(name = "products")
+@Getter
+@Setter
+@ToString
+public class Product {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    private String title;
+    private BigDecimal price;
+    @OneToOne(fetch = FetchType.LAZY, cascade = CascadeType.PERSIST)
```

Comment:
One product can have one category, but in one category could be many products. Better use `@ManyToOne` annotation here